habitat/plan.sh

```diff
@@ -78,6 +78,11 @@ do_build() {
   export GEM_PATH=${_bundler_dir}:${GEM_HOME}
 
   export NOKOGIRI_CONFIG="--use-system-libraries --with-zlib-dir=${_zlib_dir} --with-xslt-dir=${_libxslt_dir} --with-xml2-include=${_libxml2_dir}/include/libxml2 --with-xml2-lib=${_libxml2_dir}/lib"
+  
+  ## Force bundler version 1.17.3
+  gem install bundler --version 1.17.3 --silent
```

Comment:
This should grab the value from omnibus_overrides so we don't accidentally end up with bundler 2 on omnibus and 1.17 on habitat.